docs/api/web-frame-main.md

```diff
@@ -26,7 +26,7 @@ win.webContents.on(
 )
 ```
 
-You can also access frames of existing pages by using the `webFrame` property
+You can also access frames of existing pages by using the `mainFrame` property
```

Comment:
should this fix be a separate PR + backported to 12.x?

Reply:
I would like to backport this whole PR to 12-x-y if possible